test/sequential/test-debugger-restart-message.js

```diff
@@ -1,30 +1,12 @@
 'use strict';
 
-// Refs: https://github.com/nodejs/node/issues/39272
-
 const common = require('../common');
 
-const assert = require('assert');
-
-// When this is moved out of known_issues, this skip can be removed.
-if (common.isOSX) {
-  assert.fail('does not fail reliably on macOS in CI');
-}
+common.skipIfInspectorDisabled();
 
-// When this is moved out of known_issues, this can be removed and replaced with
-// the commented-out use of common.skipIfInspectorDisabled() below.
-if (!process.features.inspector) {
-  assert.fail('Known issues test should fail, so if the inspector is disabled');
-}
-
-// Will need to uncomment this when moved out of known_issues.
-// common.skipIfInspectorDisabled();
+const assert = require('assert');
 
-// This can be reduced to 2 or even 1 (and the loop removed) once the debugger
-// is fixed. It's set higher to make sure that the error is tripped reliably
-// in CI. On most systems, the error will be tripped on the first test, but
-// on a few platforms in CI, it needs to be many times.
-const RESTARTS = 16;
+const RESTARTS = 10;
```

Comment:
How we came up with this number? Is it based on the failure of test assertion?

Reply:
It won't fail reliably with just one attempt so I tried 10 and that worked. I didn't do any further analysis. I could put it back to 16 but that was equally non-scientific. (I know because I chose that number too.)

Reply:
> (I know because I chose that number too.)
😅 nice